index.d.ts

```diff
@@ -0,0 +1,8 @@
+import {ParseResult} from '@babel/parser';
+import {File} from '@babel/types';
+import type {Module} from '@swc/core';
+
+declare module 'swc-to-babel' {
+  function toBabel(ast: Module, src: string): ParseResult<File>;
+  export default toBabel;
+}
```

Comment:
Could we put this file near `swc-to-babel.js` with a name `swc-to-babel.d.ts`?

Reply:
I don't think it should go in there

Reply:
> Could we put this file near `swc-to-babel.js` with a name `swc-to-babel.d.ts` in a `lib` directory?

I did it